src/client/components/opt-in/opt-in.vue

```diff
@@ -0,0 +1,79 @@
+<template>
+  <div
+    v-if="!isAccepted"
+    role="dialog"
+    class="opt-in"
+    aria-labelledby="opt-in__title"
+    aria-describedby="opt-in__text"
+  >
+    <h2 class="a11y-sr-only" >{{title}}</h2>
+    <div id="opt-in__text" class="opt-in__text" v-html="body"></div>
+    <div class="opt-in__buttons">
+      <button
+        class="button button--primary"
+        @click="onAccept">
+        {{ $t('agree') }}
+      </button>
+      <button
+        class="button"
+        @click="onDecline"
+      >
+        {{ $t('decline') }}
+      </button>
+    </div>
+  </div>
+</template>
+
+<script>
+export default {
+  data () {
+    let isAccepted = 0
+    try {
+      isAccepted = localStorage.getItem(this.name)
+    } catch (e) {
+      isAccepted = 0
+    }
+    return { isAccepted }
+  },
+  props: {
```

Comment:
`props` before `data`? They are what goes in before data is even defined. And if something is already defined as prop, Vue throws an error when trying to set it again in data.